docs/flask.md

```diff
@@ -0,0 +1,118 @@
+# Flask-GraphQL
+
+Adds GraphQL support to your Flask application.
+
+## Installation
+
+To install the integration with Flask, run the below command on your terminal.
+
+`pip install graphql-server-core[flask]`
+
+## Usage
+
+Use the `GraphQLView` view from `graphql_server.flask`.
+
+```python
+from flask import Flask
+from graphql_server.flask import GraphQLView
+
+from schema import schema
+
+app = Flask(__name__)
+
+app.add_url_rule('/graphql', view_func=GraphQLView.as_view(
+    'graphql',
+    schema=schema,
+    graphiql=True,
+))
+
+# Optional, for adding batch query support (used in Apollo-Client)
+app.add_url_rule('/graphql/batch', view_func=GraphQLView.as_view(
```

Comment:
Not relevant to this PR but there is an open issue on graphene-django to allow the view to detect if a request is a batch request instead of having to define a separate endpoint for it: https://github.com/graphql-python/graphene-django/issues/967

We should consider doing the same in graphql-server.